utils/badPractices.ts

```
@@ -0,0 +1,114 @@
+  password,
+  apiKey
+) {
+  // Multiple issues in one function
+  
+  // Logging sensitive data
+  console.log('SSN:', socialSecurityNumber)
+  console.log('Credit Card:', creditCardNumber)
+  console.log('Password:', password)
+  
+  // Using var instead of let/const
+  var result: any = {}
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)